src/Main.cpp

```diff
@@ -5,170 +5,126 @@
 
 using namespace core;
 
-struct PadCollisionSignal {
-	ecs::Entity ball;
-	ecs::Entity pad;
-};
-
-struct Player {};
-
-struct Ball 
+struct Movement 
 {
-	geometry::Vec2 speed;
+	geometry::Vec2 velocity;
+	geometry::Vec2 velocitymax;
+	geometry::Vec2 force;
+	F32            moveforce;
+
+	Movement(F32 moveforce, F32 velocitymax) : velocity(0.0f, 0.0f), moveforce(moveforce), velocitymax(velocitymax), force(0.0f, 0.0f) { }
 };
 
 struct KeyBindings 
 {
-	KeyCode up;
-	KeyCode down;
+	KeyCode left, down, up, right;
 };
 
-struct PhysicsSystem
-	: public ecs::System
-	, public AccessGroupStorage<Player, Position>
-	, public AccessGroupStorage<Ball, Position>
-	, public SignalEmitter<PadCollisionSignal>
+static float fsignf(float x)
```

Comment:
Could we put all of these functions into a header? Also, why are these all `static`?